avaframe/in3Utils/cfgUtils.py

```diff
@@ -264,17 +266,31 @@ def writeCfgFile(avaDir, module, cfg, fileName=''):
     # write to file
     if fileName != '':
         # set outputs
-        outDir = pathlib.Path(avaDir, 'Outputs', modName, 'configurationFiles')
-        fU.makeADir(outDir)
+        if filePath == '':
+            outDir = pathlib.Path(avaDir, 'Outputs', modName, 'configurationFiles')
```

Comment:
This is different from line 284, why? Did we not say that all configuration files go to the module configuration file folder?

Reply:
I added the option to save this somewhere else, like here we want to save it to the work folder of the module as the com1DFA settings file is updated inside the loop to be used for com1DFA and overwritten - but not needed anymore after computation as the configuration for com1DFA itself has been saved with the outputs